src/udev/scsi_id/scsi_id.c

```diff
@@ -304,6 +310,18 @@ static int set_options(int argc, char **argv,
                         export = true;
                         break;
 
+                case 'i':
+                        if (arg_scsi_interface != SCSI_ID_INTERFACE_UNKNOWN)
+                                return log_error_errno(SYNTHETIC_ERRNO(EINVAL), "Invalid Interface");
```

Comment:
Please make the log message more descriptive, e.g. `"--sgio/-i/--sysfs/-y can be specified only once."`

Reply:
Done